andela/tsunade-cfh
# chore(feature): modify events and functions for czar to draw cards

### What does this PR do?
This PR allows the Czar to draw the next question by clicking on a deck of cards.
### Description of Task to be completed?
Created functions, event listeners and card deck to listen to the different states of the Czar
### How should this be manually tested?
Try to sign in as a player or play as guest and you will notice a deck of cards for you to draw question for the next round given that you are the Czar 
### Any background context you want to provide?
No
### What are the relevant pivotal tracker stories?
CZARs should be able to draw cards by clicking   #137090935
### Screenshots (if appropriate)
<img width="1222" alt="screen shot 2017-02-10 at 6 56 49 pm" src="https://cloud.githubusercontent.com/assets/24937779/22838125/f49190de-efc3-11e6-8dbb-ef55d7eb53cc.png">
### Questions:
No
